release/archives/linux/data-prepper-jdk-x64.sh

```diff
@@ -25,4 +15,4 @@ echo "JAVA_HOME is set to $JAVA_HOME"
 export PATH=$JAVA_HOME/bin:$PATH
 
 DATA_PREPPER_JAVA_OPTS="-Dlog4j.configurationFile=$DATA_PREPPER_HOME/config/log4j2-rolling.properties"
-java $JAVA_OPTS $DATA_PREPPER_JAVA_OPTS -cp "$DATA_PREPPER_CLASSPATH" org.opensearch.dataprepper.DataPrepperExecute $PIPELINES_FILE_LOCATION $CONFIG_FILE_LOCATION
\ No newline at end of file
+java $JAVA_OPTS $DATA_PREPPER_HOME_OPTS $DATA_PREPPER_JAVA_OPTS -cp "$DATA_PREPPER_CLASSPATH" org.opensearch.dataprepper.DataPrepperExecute
```

Comment:
I don't see this being set in this script.

Reply:
Fixed in [46460e9](https://github.com/opensearch-project/data-prepper/pull/1737/commits/46460e974fdcf536de2fb91782fb59463ff32169)